File: Level-2/18.java
class Solution {
    public int[][] answer;
    public int x = 0;
    public void hanoi(int num, int from, int other, int to) {
        if(num == 0) return;
        hanoi(num - 1, from, to, other);
        answer[x][0] = from;
        answer[x++][1] = to;
        hanoi(num - 1, other, from, to);
    }
    public int[][] solution(int n) {
        int size = (int) Math.pow(2, n) - 1;
        answer = new int[size][2];
        hanoi(n, 1, 2, 3);
        return answer;
    }
}

File: Level-2/18.js
function solution(s) {
    const answer = s.split(' ').map(el => {
        let s = '';
        for(let i = 0; i < el.length; i++) {
            if(i === 0) s += el[i].toUpperCase();
            else s += el[i].toLowerCase();
        }
        return s;
    });
    
    return answer.join(' ');
}
